ecommerce-ui/src/VacayRentalContainer.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import VacayRentalContainer from './VacayRentalContainer';
+import { isFunction } from 'util';
+
+it('shallow renders VacayRentalContainer', () => {
+    const wrapper = shallow(
+    <VacayRentalContainer
+        onBookListing = {isFunction}
```

Comment:
You can also use `jest.fn()` instead of `isFunction` to pass in a function for this test. Either technically works.